aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -441,6 +445,29 @@ mod loader {
             self
         }
 
+        /// When true, send this request to the FIPS-compliant regional endpoint.
+        ///
+        /// If the configured endpoint does not have a FIPS compliant endpoint, dispatching
+        /// the request will return an error.
+        ///
+        /// **Note**: Not all services and regions support FIPS. If a service does not support FIPS,
+        /// this setting will have no effect.
+        pub fn use_fips(mut self, use_fips: bool) -> Self {
+            self.use_fips = Some(use_fips);
+            self
+        }
+
+        /// When true, send this request to the dual-stack endpoint.
+        ///
+        /// If the configured endpoint does not support dual-stack, the request MAY return an error.
+        ///
+        /// **Note**: Not all services and regions support dual-stack. If a service does not support
+        /// dual-stack, this setting will have no effect.
```

Comment:
Same here, I'm not clear on the difference between an endpoint supporting dual-stack and a service supporting dual-stack